### ISSUE TITLE
Resident Evil Revelations 2

### ISSUE THREAD

**Issue by nigelweng** [Electry/VitaGrafix#52]
Hello Electry, Is it possible to add support for this game? 
So i can reduce resolution to make it smooth.
Thank you for all your hard work.

**Comment by ChiSys** [comment]
Thought about the same.

**Comment by SheidCraft** [comment]
In the first version of the game, the frame rate was almost stable, but with the patch fps greatly decreased. Is it possible to start the game with the original configuration, but with the raid mode?

**Comment by Electry** [comment]
@gittyno1 As it can't even hold 30fps, 60fps just won't happen.

**Comment by Caliburnflash** [comment]
Is there a chance to somehow unlock the framerate?

**Comment by SheidCraft** [comment]
Yes unlock the frequency not the problem of :) the problem is that пропатченая game not reaches and 30fps. of course overclock helps with framerate, but there is still strong subsidence. although I would взлянул on the game in a little better resolution (408), because I think there 368p, but information about this I find I can not

**Comment by yeqing1990** [comment]
This game has been able to 960x544, after the landlord modified the TBOOT file.

**Comment by SheidCraft** [comment]
The original version without a patch is quite stable and with overclocking it can even pull 544p, and 408p
